src/models/IndexedMatterGram.ts

```diff
@@ -0,0 +1,39 @@
+import { ResponseData } from "../networking/NetworkResponse";
+import { DocumentReference } from "@firebase/firestore";
+import Matter from "./Matter";
+import { Model } from "./Model";
+
+class IndexedMatterGram implements Model {
+  id: string;
+  matter_ref: string;
+  matter?: Matter;
+  unstemmed_gram: string;
+  stemmed_gram: string;
+  context_span: string;
+  value: number;
+  datetime_weighted_value: number;
+  keywords?: string[];
+
+  constructor(jsonData: ResponseData) {
+    this.id = jsonData["id"];
+    this.unstemmed_gram = jsonData["unstemmed_gram"];
+    this.stemmed_gram = jsonData["stemmed_gram"];
+    this.context_span = jsonData["context_span"];
+    this.value = jsonData["value"];
+    this.datetime_weighted_value = jsonData["datetime_weighted_value"];
+    this.matter_ref = jsonData["matter_ref"].id;
+
+    if (
+      typeof jsonData["matter_ref"] === "object" &&
+      !(jsonData["matter_ref"] instanceof DocumentReference)
+    ) {
+      this.matter = new Matter(jsonData["event_ref"]);
+    }
+
+    if (jsonData["keywords"]) {
+      this.keywords = jsonData["keywords"];
+    }
```

Comment:
`IndexedMatterGram` are the keywords, at least the top 5, of a `Matter`. I don't think `.keywords` would be a field on `IndexedMatterGram`.

cc @JacksonMaxfield

Reply:
Yes the "keywords" prop should be removed but to be fair, looks like the mocked models dont implement the keywords prop :shrug: 

My opinion is its okay to leave as is because after I finish the matter index pipeline I can come update this class too.

Reply:
Right - after talking with Jackson yesterday, it seems the "keyword" is the unstemmed gram property. The "keywords" list then, is simply a list of the unstemmed grams from an array of IndexedMatterGrams.

Removed this wrong property.